### ISSUE TITLE
statements/generators/params-trailing-comma-length.js has incorrect generator syntax

### ISSUE THREAD

**Issue by JosephPecoraro** [tc39/test262#761]
Test: `test/statements/generators/params-trailing-comma-length.js` has incorrect syntax:

``` javascript
function one*(a,) {}
function two*(a,b,) {}
```

It should be:

``` javascript
function* one(a,) {}
function* two(a,b,) {}
```

**Comment by rwaldron** [comment]
@JosephPecoraro please sign the CLA: http://tc39.github.io/test262-cla/ and ping back here when that's complete. Thanks!!

**Comment by leobalter** [comment]
This is fixed on #762
